core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.ts

```diff
@@ -241,6 +241,17 @@ export class LessonInformationCardModalComponent extends ConfirmOrCancelModal {
 
   copyProgressUrl(): void {
     this.clipboard.copy(this.loggedOutProgressUniqueUrl);
+    const messArrow = document.getElementsByClassName('uid-btn-message-arrow');
+    const messBox = document.getElementsByClassName('uid-btn-message-box');
+    if(messArrow[0]==undefined || messBox[0]==undefined){
+      return ;
+    }
+    messArrow[0].classList.add('d-block');
+    messBox[0].classList.add('d-block');
+    setTimeout(() => {
+      messArrow[0].classList.remove('d-block');
+      messBox[0].classList.remove('d-block');
+    }, 1000);
```

Comment:
Can we just not simply use `*ngIf` in html instead of using d-none and d-block and just change the variable to true on clicking 'Copy' button and use setTimeout to set it back to false ?

Reply:
done using ngIf. thanks for telling!